### PR TITLE
Update README.zh-CN.md

### PR DESCRIPTION
**Pull request by subtank**
job done~

### CODE REVIEW

File: README.zh-CN.md [hunk 1]
```diff
@@ -18,13 +18,20 @@
  - [X] 可扩展空岛领地范围（需要付出一些资源/金钱需要经济插件支持）
  - [X] 可单独创建一个空岛世界与其他世界分离
  - [X] 自定义岛屿重生点或者tp点
- - [X] 利用GriefPrevention的/trust命令来创建一个空岛队伍
+ - [X] 团队功能：
```
Comment (subtank) [inline README.zh-CN.md:21]: still some format problem.why?

Comment (Mohron) [reply README.zh-CN.md:21]: What do you mean?

File: README.zh-CN.md [hunk 2]
```diff
@@ -18,13 +18,20 @@
...
+   - [X] 邀请朋友到岛上
+   - [X] 踢出一个在岛上的玩家
+   - [X] 离开一个岛
+   - [X] 转赠一个岛屿(移交财产么?2333)
+   <!--did your mean it is the dead money? -->
```
Comment (Mohron) [inline README.zh-CN.md]: I have no idea what you are saying here.

**Comment by Mohron** [comment]
Can you target the feature/invite-system branch?

File: README.zh-CN.md (hunk 1 above)
Comment (subtank) [reply README.zh-CN.md:21]: view page did not work as my thought.

File: README.zh-CN.md (hunk 2 above)
Comment (subtank) [reply README.zh-CN.md]: did you mean the island like something giveaway?or inheritance?

File: README.zh-CN.md (hunk 1 above)
Comment (Mohron) [reply README.zh-CN.md:21]: Four spaces before the dash

Comment (subtank) [reply README.zh-CN.md:21]: thanks

File: README.zh-CN.md (hunk 2 above)
Comment (Mohron) [reply README.zh-CN.md]: Islands have a single owner who has full control over the said island. Transfering ownership means giving that control to somebody else.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (subtank) [reply README.zh-CN.md]: and than the new owner kicked you out?ROFL.

Comment (Mohron) [reply README.zh-CN.md]: @subtank more than likely if you're transferring ownership, you intend to leave the island.

File: README.zh-CN.md [hunk 3]
```diff
@@ -18,13 +18,20 @@
  - [X] 可扩展空岛领地范围（需要付出一些资源/金钱需要经济插件支持）
  - [X] 可单独创建一个空岛世界与其他世界分离
  - [X] 自定义岛屿重生点或者tp点
- - [X] 利用GriefPrevention的/trust命令来创建一个空岛队伍
+ - [X] 团队功能：
+    - [X] 邀请朋友到岛上
+    - [X] 踢出一个在岛上的玩家
+    - [X] 离开一个岛
+    - [X] 转赠一个岛屿(移交财产么?2333)
```
Comment (Mohron) [inline README.zh-CN.md:25]: Do these numbers mean something?

Comment (subtank) [reply README.zh-CN.md:25]: chinese LOL

Comment (Mohron) [reply README.zh-CN.md:25]: ?2333 is Chinese ??

Comment (subtank) [reply README.zh-CN.md:25]: ROFL LOL or oher in laugh

Comment (Mohron) [reply README.zh-CN.md:25]: What kind of translation are you doing LMAO

Comment (subtank) [reply README.zh-CN.md:25]: you add this features let me think it is like: i will dead few mins later,this island give to you and pls take care of it.

Comment (subtank) [reply README.zh-CN.md]: you add this features let me think it is like: i will dead few mins later,this island give to you and pls take care of it.

Comment (subtank) [reply README.zh-CN.md]: you add this features let me think it is like: i will dead few mins later,this island give to you and pls take care of it.

Comment (subtank) [reply README.zh-CN.md]: you add this features let me think it is like: i will dead few mins later,this island give to you and pls take care of it.

Comment (Mohron) [reply README.zh-CN.md:25]: So you're sharing your delight with your fellow Chinese users? I see :p

**Pull request merged**